init/init_raphael.cpp

```diff
@@ -49,4 +49,8 @@ void vendor_load_properties() {
         load_raphaelglobal();
     else if (region.find("INDIA") != std::string::npos)
         load_raphaelin();
+
+    if (region.find("CN") != std::string::npos ||
+        region.find("GLOBAL") != std::string::npos)
```

Comment:
Maybe it could be simpler?
```
if (region.find("INDIA") == std::string::npos)
```

Reply:
> Maybe it could be simpler?
> 
> ```
> if (region.find("INDIA") == std::string::npos)
> ```

Well we are setting sku's for China and Global variants codenamed "raphael" and this seems to be easier to start nfc service instead of adding sku for just disabling them for Indian variant

Reply:
If no matches were found, the function returns string::npos.
Ref: https://www.cplusplus.com/reference/string/string/find/

Reply:
Note that I wrote `==` and not `!=`.